lib/async.js

```diff
@@ -5,8 +5,6 @@ var nodeModulesPaths = require('./node-modules-paths.js');
 var normalizeOptions = require('./normalize-options.js');
 var isCore = require('./is-core');
 
-var realpath = typeof fs.realpath.native === 'function' ? fs.realpath.native : fs.realpath;
```

Comment:
The solution is *not* to make this module brittle by not caching it at require time; it's to not crash when `fs.realpath` is nullish.